src/System.Management.Automation/engine/SessionStateVariableAPIs.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
```

Comment:
@iSazonov, your last commit had 19 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Validate that get-help works for AllUsers Scope.for module : .Error occurred in Context block
```powershell
The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The following parameter must not be null or empty: Module.
At /Users/vsts/agent/2.153.2/work/1/s/test/powershell/engine/Help/HelpSystem.Tests.ps1:31 char:5
```
help can be found for AllUsers Scope.help in user scope be found for <TestName>
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/engine/Help/HelpSystem.Tests.ps1: line 501
501:         $helpObj = Get-Help -Name $CmdletName -Full

```
Basic SysLog tests on Linux.Error occurred in Describe block
```powershell
The variable '$IsSupportedEnvironment' cannot be retrieved because it has not been set.
At /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/Logging.Tests.ps1:171 char:64
```
Basic os_log tests on MacOS.Error occurred in Describe block
```powershell
The variable '$IsSupportedEnvironment' cannot be retrieved because it has not been set.
At /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/Logging.Tests.ps1:309 char:64
```
Verify sub-directory creation under root.Can create a sub directory under root path
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1425
1425:         New-Item -Path $dirPath -ItemType Directory -Force > $null

```